src/lib/Makefile.am

```diff
@@ -20,7 +20,6 @@ librnp_la_SOURCES	= \
 	key_store.c \
 	key_store_pgp.c \
 	key_store_kbx.c \
-	key_store_ssh.c \
```

Comment:
Why you just dissable it? Have we got any discussion about it tat I missed?

Anyway I see your question at Ribose and it was created 10 minutes ago when this comment was created 40 minutes...